homeassistant/components/frontend/__init__.py

```diff
@@ -357,8 +357,10 @@ async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
     )
 
     local = hass.config.path("www")
-    if os.path.isdir(local):
-        hass.http.register_static_path("/local", local, not is_dev)
```

Comment:
Why don't we just always register a local path, regardless if the dir exists? Then if it's created it automatically works.

Reply:
That didn't work in my testing.